drivers/video/video_common.c

```diff
@@ -99,6 +101,8 @@ void video_buffer_release(struct video_buffer *vbuf)
 int video_format_caps_index(const struct video_format_cap *fmts, const struct video_format *fmt,
 			    size_t *idx)
 {
+	__ASSERT_NO_MSG(fmts != NULL && fmt != NULL && idx != NULL);
```

Comment:
I didn't think of this last time I reviewed, but it is better to split these. Otherwise you don't know which assert failed.

```suggestion
	__ASSERT_NO_MSG(fmts != NULL);
	__ASSERT_NO_MSG(fmt != NULL);
	__ASSERT_NO_MSG(idx != NULL);
```